randovania/games/am2r/json_data/Golden Temple.txt

```diff
@@ -762,7 +766,7 @@ Extra - map_name: rm_a1a08
                   All of the following:
                       # Spider requires another power bomb, or a way to spider at the top
                       Can Use Spider Ball
-                      Mid-Air Morph (Beginner) or Can Use Power Bombs or Can Use Spring Ball
+                      Power Bombs ≥ 2 or Mid-Air Morph (Beginner) or Can Use Spring Ball
```

Comment:
I'd personally change the comment to say "requires two pbs" instead of "requires another pb", just to hopefully limit the amount of people who also think ammo counts are accumulative.

Reply:
Sure thing. Will update.